src/main/java/fr/kungfunantes/backend/security/AuthResource.java

```diff
@@ -82,6 +82,7 @@
         response.put("token", new JwtAuthenticationResponse(jwt));
         response.put("user", (UserPrincipal) authentication.getPrincipal()); // get signed in user
 
+
```

Comment:
not a real problem to have this, but it's symptomatic that you don't review what you are committing. And that is a bah habit.
A commit should be like when you're giving back an exam: you take all the pages you have written down, you go over it quickly to ensure you are not giving back your notes nor your draft, and that everything is readable.
That would avoid uselessly committing files